### PR TITLE
modify prepare-swagger.sh and configure_swagger.md

### PR DESCRIPTION
**Pull request by wemeya**

### CODE REVIEW

File: docs/configure_swagger.md [hunk 1]
```diff
@@ -48,25 +46,20 @@ From time to time, you may need to mannually test Harbor REST API. You can deplo
...
-    docker-compose build
-```
-* Clean up the previous running version. NOTE: The command does not remove your existing data.
-```docker
-   docker-compose rm
-```
-* Start the new Harbor build
-```docker
-   docker-compose up
+    docker-compose down & docker-compose up -d
```
Comment (reasonerjt) [inline docs/configure_swagger.md]: better use "docker-compose down -v"

Comment (wemeya) [inline docs/configure_swagger.md]: modified.

**Comment by wemeya** [comment]
@hainingzhang already changed as you required, pls have a review.

File: docs/configure_swagger.md [hunk 2]
```diff
@@ -1,24 +1,22 @@
...
+
 ### Viewing Harbor REST API
 * Open the file **swagger.yaml** under the _docs_ directory in Harbor project;
 * Paste all its content into the online Swagger Editor at http://editor.swagger.io. The descriptions of Harbor API will be shown on the right pane of the page.
 
 ![Swagger Editor](img/swaggerEditor.png)
 
 ### Testing Harbor REST API
-From time to time, you may need to mannually test Harbor REST API. You can deploy the Swagger file into Harbor's service node. 
+From time to time, you may need to mannually test Harbor REST API. You can deploy the Swagger file into Harbor's service node. We suppose you install Harbor through online or offline installer, then you should have a Harbor directory after you untar the installer, such as **~/harbor**.
```
Comment (hainingzhang) [inline docs/configure_swagger.md]: Suppose you install Harbor through the online or offline installer, then you should have a Harbor directory after you un-tar the installer,

File: docs/configure_swagger.md [hunk 3]
```diff
@@ -1,24 +1,22 @@
...
 
 **Caution:** When using Swagger to send REST requests to Harbor, you may alter the data of Harbor accidentally. For this reason, it is NOT recommended using Swagger against a production Harbor instance.
 
-* Change the directory to _docs_ in Harbor project.
+* Download _prepare-swagger.sh_ and _swagger.yaml_ under the _docs_ directory to your local Harbor directory, eg: **~/harbor**.
```
Comment (hainingzhang) [inline docs/configure_swagger.md]: eg: --> e.g.

Comment (wemeya) [inline docs/configure_swagger.md]: updated.

**Pull request merged**